# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/targets/translator/moz.build
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/targets/translator/moz.build
--- a/gfx/angle/targets/translator/moz.build	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/targets/translator/moz.build	Sat Sep 07 13:38:36 2019 +0000
@@ -13,7 +13,7 @@
 #DEFINES['FULL_SAFE_BROWSING'] = True
 DEFINES['NOMINMAX'] = True
 #DEFINES['NO_TCMALLOC'] = True
-DEFINES['NTDDI_VERSION'] = '0x0A000003'
+DEFINES['NTDDI_VERSION'] = 'NTDDI_WIN10_RS2'
 #DEFINES['PSAPI_VERSION'] = '2'
 #DEFINES['SAFE_BROWSING_CSD'] = True
 #DEFINES['SAFE_BROWSING_DB_LOCAL'] = True
@@ -47,7 +47,6 @@
 
 #CXXFLAGS += [
 #    '/bigobj',
-#    '/d2FastFail',
 #    '/D__DATE__=',
 #    '/D__TIME__=',
 #    '/D__TIMESTAMP__=',
@@ -106,6 +105,7 @@
 #    '/we4244',
 #    '/we4456',
 #    '/we4458',
+#    '/we4715',
 #    '/we4800',
 #    '/we4838',
 #    '/WX',
@@ -176,8 +176,8 @@
     '../../checkout/src/compiler/translator/tree_ops/ClampPointSize.cpp',
     '../../checkout/src/compiler/translator/tree_ops/DeclareAndInitBuiltinsForInstancedMultiview.cpp',
     '../../checkout/src/compiler/translator/tree_ops/DeferGlobalInitializers.cpp',
-    '../../checkout/src/compiler/translator/tree_ops/EmulateGLDrawID.cpp',
     '../../checkout/src/compiler/translator/tree_ops/EmulateGLFragColorBroadcast.cpp',
+    '../../checkout/src/compiler/translator/tree_ops/EmulateMultiDrawShaderBuiltins.cpp',
     '../../checkout/src/compiler/translator/tree_ops/EmulatePrecision.cpp',
     '../../checkout/src/compiler/translator/tree_ops/ExpandIntegerPowExpressions.cpp',
     '../../checkout/src/compiler/translator/tree_ops/FoldExpressions.cpp',
@@ -193,7 +193,9 @@
     '../../checkout/src/compiler/translator/tree_ops/RemovePow.cpp',
     '../../checkout/src/compiler/translator/tree_ops/RemoveSwitchFallThrough.cpp',
     '../../checkout/src/compiler/translator/tree_ops/RemoveUnreferencedVariables.cpp',
+    '../../checkout/src/compiler/translator/tree_ops/RewriteAtomicCounters.cpp',
     '../../checkout/src/compiler/translator/tree_ops/RewriteAtomicFunctionExpressions.cpp',
+    '../../checkout/src/compiler/translator/tree_ops/RewriteDfdy.cpp',
     '../../checkout/src/compiler/translator/tree_ops/RewriteDoWhile.cpp',
     '../../checkout/src/compiler/translator/tree_ops/RewriteElseBlocks.cpp',
     '../../checkout/src/compiler/translator/tree_ops/RewriteExpressionsWithShaderStorageBlock.cpp',
@@ -214,11 +216,13 @@
     '../../checkout/src/compiler/translator/tree_ops/UseInterfaceBlockFields.cpp',
     '../../checkout/src/compiler/translator/tree_ops/VectorizeVectorScalarArithmetic.cpp',
     '../../checkout/src/compiler/translator/tree_ops/WrapSwitchStatementsInBlocks.cpp',
+    '../../checkout/src/compiler/translator/tree_util/FindFunction.cpp',
     '../../checkout/src/compiler/translator/tree_util/FindMain.cpp',
     '../../checkout/src/compiler/translator/tree_util/FindSymbolNode.cpp',
     '../../checkout/src/compiler/translator/tree_util/IntermNode_util.cpp',
     '../../checkout/src/compiler/translator/tree_util/IntermNodePatternMatcher.cpp',
     '../../checkout/src/compiler/translator/tree_util/IntermTraverse.cpp',
+    '../../checkout/src/compiler/translator/tree_util/ReplaceShadowingVariables.cpp',
     '../../checkout/src/compiler/translator/tree_util/ReplaceVariable.cpp',
     '../../checkout/src/compiler/translator/tree_util/RunAtTheEndOfShader.cpp',
     '../../checkout/src/compiler/translator/Types.cpp',
